src/classes/CMT_FilterRule.cls

```diff
@@ -120,8 +121,9 @@ public class CMT_FilterRule {
         this.field = UTIL_Describe.getFieldDescribe(filterRule.Object__r.QualifiedApiName, filterRule.Field__r.QualifiedApiName).getSObjectField();
         this.constant = filterRule.Constant__c;
         this.operation = CMT_FilterRuleUI_SVC.getFilterOperationFromString(filterRule.Operator__c);
+        this.fieldType = field.getDescribe().getSoapType();
 
-        if (field.getDescribe().getSOAPType() == SoapType.DATETIME && constant != null &&
+        if (fieldType == SoapType.DATETIME && constant != null &&
```

Comment:
Should we update the comment below to reflect that the UI will not allow for datetime and will only accept dates?